src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingCtorCheckTest.java

```diff
@@ -69,7 +69,7 @@ public void testMissingCtorClassOnOneLine() throws Exception {
         };
 
         verifyWithInlineConfigParser(
-                getPath("InputMissingCtor2.java"),
+                getPath("InputMissingCtorNestedClasses.java"),
```

Comment:
@CubeTures ,
The issue requires renaming of test method names along with the java input files.
Please do change test name above : 

testMissingCtorClassOnOneLine() --> `testMissingCtorNestedClasses`

Reply:
Thanks. I'm new to using git so I had actually made two commits instead of just the one. In the second commit I had renamed the test method name as well. Thanks for your help and I'll be sure to find a new name for the original InputMissingCtor.java as well!